tests-invoke

```diff
@@ -268,7 +268,7 @@ class PullTask(object):
                 else:
                     time.sleep(60)
             ret = p.returncode
-            sys.stderr.write("Test run finished, return code: {0}\n", ret)
+            sys.stderr.write("Test run finished, return code: {0}\n", str(ret))
```

Comment:
No, it needs to call `.format(ret)`, or use `'code: %s' % ret`. The stringification is totally not the problem.

Reply:
sorry my brain is fried :|